net/kcptun/Makefile

```diff
@@ -1,13 +1,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=kcptun
-PKG_VERSION:=20210922
-PKG_RELEASE:=$(AUTORELEASE)
+PKG_VERSION:=20230207
+PKG_RELEASE:=2
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://codeload.github.com/xtaci/kcptun/tar.gz/v${PKG_VERSION}?
-PKG_SOURCE_DATE:=2021-09-22
-PKG_HASH:=f6a08f0fe75fa85d15f9c0c28182c69a5ad909229b4c230a8cbe38f91ba2d038
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
```

Comment:
uh? why?

Reply:
@1715173329 
because the latest release version cannot be found in this PKG_SOURCE_URL

Reply:
```console
SHELL= flock /home/immortalwrt/openwrt-master/tmp/.kcptun-20230207.tar.gz.flock -c '  	/home/immortalwrt/openwrt-master/scripts/download.pl "/home/immortalwrt/openwrt-master/dl" "kcptun-20230207.tar.gz" "skip" "" "https://codeload.github.com/xtaci/kcptun/tar.gz/v20230207?"    '
+ curl -f --connect-timeout 20 --retry 5 --location https://codeload.github.com/xtaci/kcptun/tar.gz/v20230207?/kcptun-20230207.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 2545k    0 2545k    0     0  2502k      0 --:--:--  0:00:01 --:--:-- 2502k
touch /home/immortalwrt/openwrt-master/build_dir/target-aarch64_generic_musl/kcptun-20230207/.prepared_4a6dd31649740a5e7fd8bb3f5dc0a447_6664517399ebbbc92a37c5bb081b5c53_check
```

seems to work for me

===

don't name PKG_SOURCE as `v$(PKG_VERSION).tar.gz` only, it's really unrecognizable and can cause (potential) conflict.
btw unused `PKG_SOURCE_DATE` can be removed.